tiatoolbox/tools/patchextraction.py

```diff
@@ -329,10 +262,8 @@ def get_patch_extractor(method_name, **kwargs):
     """
     if method_name.lower() == "point":
         patch_extractor = PointsPatchExtractor(**kwargs)
-    elif method_name.lower() == "fixedwindow":
-        patch_extractor = FixedWindowPatchExtractor(**kwargs)
-    elif method_name.lower() == "variablewindow":
-        patch_extractor = VariableWindowPatchExtractor(**kwargs)
+    elif method_name.lower() == "sliding":
```

Comment:
Can we call it `slidingwindow` instead of `sliding` ?

Reply:
resolved in d1bfe24cdc0100a5b1a855be87ab8adb100a7926